.github/workflows/build-and-test.yml

```diff
@@ -55,6 +55,30 @@ jobs:
       - name: goimpi
         run: make goimpi
 
+  govulncheck:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
```

Comment:
Can you bump this up to 30 mins?  We have this workflow in Contrib and if it ever can't grab the cache correctly and has to redownload everything it can run long.

Reply:
changed